include/srtp.h

```diff
@@ -216,6 +216,7 @@ typedef enum {
     srtp_err_status_pkt_idx_adv = 27,   /**< packet index advanced, reset    */
                                         /**< needed                          */
     srtp_err_status_buffer_small = 28,  /**< out buffer is too small         */
+    srtp_err_status_cryptex_err = 29,   /**< unsupported cryptex operation   */
```

Comment:
Rather than adding this error, why not just disable extension encryption of extensions are not present?

Reply:
Want to avoid people thinking the csrc's will be protected but are actually not as they did not include a header extension, unless I misunderstood you comment.

Reply:
OK, that's a valid argument. If the only reason for the error is that extension encryption is requested, but extensions were not given, perhaps that should be in the description. Like, "extensions required for cryptex".